frontend/src/components/LoggedUser.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+
+
+class LoggedUser extends Component{
+	constructor(props){
+		super(props);
+		this.state = {  }
+	}
+
+
+}
+
+//how much long has user been loggedin?
+
+export const checkTime = () => {
+	let countHours = 0;
+	if (localStorage.hasOwnProperty("isLoggedIn") && localStorage.hasOwnProperty("time")){
+		const getcurrentTime = new Date().getHours();
```

Comment:
A date object in Javascript in the number of milliseconds that have passed since January 1, 1970. You can directly subtract the two dates and then convert milliseconds to hours, see this post for reference : https://stackoverflow.com/questions/19225414/how-to-get-the-hours-difference-between-two-date-objects/19225463